### ISSUE TITLE
VCF file download not compatible with vcftools 

### ISSUE THREAD

**Issue by isaak** [solgenomics/sgn#3857]
Examples:

vcftools --vcf BreedBaseGenotypesDownload.vcf
returns an error Error: Expected at least 4 parts in INFO definition: ID=VCFDownload,Description='VCFv4.2 FILE GENERATED BY BREEDBASE AT 2021-07-23_16:34:05'>

bcftools query --list-samples BreedBaseGenotypesDownload.vcf
returns another relevant error:
W::bcf_hdr_register_hrec] An INFO field has no Type defined. Assuming String
[W::bcf_hdr_register_hrec] An INFO field has no Number defined. Assuming '.'
[E::bcf_hdr_add_sample_len] Duplicated sample name 'TMEB419'
Failed to read from /Users/mw489/Downloads/BreedBaseGenotypesDownload.vcf: could not parse header

With the 5 sample VCF file (no duplicates), the bcftools query --list-samples correctly returns the list of samples, it still gives a bit of warning about the header:
[W::bcf_hdr_register_hrec] An INFO field has no Type defined. Assuming String
[W::bcf_hdr_register_hrec] An INFO field has no Number defined. Assuming '.'
But bcftools often complains.

**Comment by ClayBirkett** [comment]
It looks like the INFO line should be removed from the header. The spec says the INFO line is used to describe in more detail the INFO column. The INFO column may include useful information but is not a requirement. So the INFO line should be supplied with the original submission file. I suggest adding a "fileDate=" line to the header and maybe change the "source=" to Breedbase, Genotype trial

**Comment by wolfemd** [comment]
Update:
`egrep -v "^##INFO=" BreedBaseGenotypesDownload\ \(2\).vcf | vcftools --vcf - ` i.e. simply removing the offending line, seems to work!
```
VCFtools - 0.1.16
(C) Adam Auton and Anthony Marcketta 2009

Parameters as interpreted:
	--vcf [stdin]

After filtering, kept 1210 out of 1210 Individuals
After filtering, kept 61239 out of a possible 61239 Sites
Run Time = 31.00 seconds
```

**Comment by isaak** [comment]
fixed #3867.

**Comment by wolfemd** [comment]
Just tried a fresh VCF download. Same accession list (IITA_ExampleGSaccessions_2021Dec05) and genotyping protocol (IITA DArt-GBS 08 Aug 2021) as before. Still have the same message from VCFtools.
